mne_bids/mne_bids.py

```diff
@@ -283,11 +317,22 @@ def _scans_tsv(raw, raw_fname, fname, verbose):
         Relative path to the raw data file.
     fname : str
         Filename to save the scans.tsv to.
+    overwrite : str, one of ('overwrite', 'error', 'clear')
+        How the file should be handled if is exists already.
+        Defaults to 'overwrite'.
+        If overwrite = `overwrite` the previously existing file will have the
+        new data added to it, overwriting any duplicate entries found.
```

Comment:
note that these docstrings are only for developers. Normally I prefer to have no docstrings for private functions unless they are really complicated. The docstring that you really need to care about is the public one. There, I don't see the exceptions for `scans.tsv` being mentioned ...

Reply:
https://github.com/mne-tools/mne-bids/blob/d501b9daf20e50e22d26504293f9cc78dd232c4c/mne_bids/mne_bids.py#L613
?
And I just added to the docstrings because they were there and I assumed that was needed.

Reply:
looks good !